doc/source/tutorials/minimum_spanning_trees/assets/minimum_spanning_trees.py

```diff
@@ -0,0 +1,35 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Generate grid graph with random weights
+np.random.seed(0)
+
+g = ig.Graph.Lattice([5, 5], circular=False)
+g.es["weight"] = np.random.randint(1, 20, g.ecount()).tolist()
```

Comment:
I'm not sure whether it's worth introducing NumPy as a dependency to this test case just to generate a few random numbers. You can achieve the same thing with `[randint(1, 20) for _ in g.es]`.

Reply:
Yep, numpy is more convenient in general, though I agree that it isn't necessary in this case, and can be avoided. I've made the changes suggested.

Reply:
Actually, this was still here, so I fixed it now.